user_guide/intro_nav_search.rst

```diff
@@ -0,0 +1,90 @@
+.. _user-guide-getting-started-search:
+
+Search
+======
+
+Search is the fastest way to find a specific record you need. To find a record in OroCRM:
+
+# Click the search icon in the top of the screen – |IcSearch|
+# Type in the search key the text field
+# Click Go or hit Enter on the keyboard
+
+
+For example, we are searching for "Ann" all over the system:
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_1.png
+
+|
+
+
+These are our search results:
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_2.png
+
+|
+
+The search returned 28 records, 2 of which are Accounts, 2 – Contacts, 6 are Magento Shopping Carts, etc. You can scroll 
+through the entire list of results or select a specific entity in order to find the desired record faster.
+
+If you wait for a few second before clicking Go, you will see a dropdown list of most relevant results. If the one you 
+are looking for is in the list, click it to open straight away. Here are the most relevant results for the example 
+above:
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_3.png
+
+|
+
+Modify the Search
+-----------------
+
+To save time, you can limit the search by a certain entity straight away. For example, if you know you’re looking for 
+an account, you can look for "Ann" in Accounts only.
+
+|
+
+.. image:: ./img/navigation/panel/search_ex_4.png
+
+|
+
+
+.. image:: ./img/navigation/panel/search_ex_5.png
+
+|
+
+
+
+.. _user-guide-getting-started-search-tag:
+
+Search by Tag
+-------------
+
+In order to see all the records with a specific tag anywhere in the system.
+
+|
+
+.. image:: ./img/navigation/panel/search_vip.png
+
+|
+
+.. image:: ./img/navigation/panel/search_vip_1.png
+
+|
+
+You will see the page similar to the search results, that will contain all the records with this tag.
```

Comment:
Edit: You will see a page that looks similar to the search results and that contains all the records with this tag.